test/aspect/aspect.bzl

```diff
@@ -53,8 +57,8 @@ def _rule_impl(ctx):
     }
     content = ""
     for target in ctx.attr.targets:
-        visited = sorted(target.visited)
-        expected = sorted(expected_deps[target.label.name])
+        visited = depset(sorted(target.visited)).to_list()
+        expected = depset(sorted(expected_deps[target.label.name])).to_list()
```

Comment:
Stupid question, is the point of doing this to dedup these lists by using the depset which will not return duplicates, and I guess the order of iteration will be identical in this case due to the sorting? Is the identical order guaranteed by depset, based on the docs wasn't fully sure.

Reply:
The default order is undefined but deterministic which IIUC gives us the equality we’re seeking for.
I think this test is obsolete but I’m not sure and would rather not delete it just this instance.

Reply:
Waiting for your 👍🏽👎🏽

Reply:
Oh I thought deterministic just meant that for a given specific input the iteration order would be the same across different calls. But I guess if it works it works.